dspace/config/spring/api/access-conditions.xml

```diff
@@ -61,4 +61,25 @@
         </property>
     </bean>
 
+    <bean id="accessConditionConfigurationService" class="org.dspace.submit.model.AccessConditionConfigurationService">
+        <property name="map">
+            <map>
+                <entry key="itemAccessConditions" value-ref="accessConditionConfigurationDefault" />
```

Comment:
Hi @benbosman  thanks for the feedback
you are right, the use of the map in the service and the name attribute in the single configuration were confusing. This was already a suboptimal implementation inherited by the upload feature, nevertheless, I have refactored the code so that the map is not longer required and only the name is used to identify the configuration. This name must match the id of the step as defined in the item-submission.xml
A similar refactoring should be done also for the upload service but it is too large to do it now so we suggest to create a code task for that in 7.3

This is the link to the test that checks if the permissions have been applied after installing the item:
https://github.com/DSpace/DSpace/pull/8091/commits/87a338b0fdec2f1ac2d7c523852823687d67b4d6#diff-6b540f70bc6b8b811121cfbbac5a52a3fb4754197bffbd675fffa7e7394b597eR7090